templates/web/common/site_layout.tt.html

```diff
@@ -51,6 +51,7 @@
 									<li><a href="[% edq(lang('menu_contribute_link')) %]">[% lang('menu_contribute') %]</a></li>
 									<li class="divider"></li>
 									<li><label>[% lang("add_products") %]</label></li>
+                <!-- For reference: get_the_app_link_off" = /open-food-facts-mobile-app" -->
 									<li><a href="[% edq(lang_flavor('get_the_app_link')) %]?utm_source=off&utf_medium=web&utm_campaign=pro_platform_install_the_app_to_add_products_[% language %]">[% lang('install_the_app_to_add_products') %]</a></li>
```

Comment:
```suggestion
									<li><a href="[% edq(lang_flavor('get_the_app_link')) %]&utm_source=off&utf_medium=web&utm_campaign=pro_platform_install_the_app_to_add_products_[% language %]">[% lang('install_the_app_to_add_products') %]</a></li>
```

Reply:
There's no question mark in the root URL, I believe @hangy

Reply:
get_the_app_link is in po files @stephanegigandet @hangy